pkg/clients/acmpca/certificateauthority.go

```diff
@@ -36,10 +36,6 @@ func GenerateCreateCertificateAuthorityInput(p *v1alpha1.CertificateAuthorityPar
 		RevocationConfiguration:           GenerateRevocationConfiguration(p),
 	}
 
-	// if p.RevocationConfiguration != nil {
-	// 	m.RevocationConfiguration = GenerateRevocationConfiguration(p.RevocationConfiguration)
-	// }
-
```

Comment:
I think you may need a rebase on `master` here.

Reply:
Rebased with master and pushed the changes.